13/umbraco-cms/reference/content-delivery-api/README.md

```diff
@@ -24,20 +24,17 @@ Video tutorial
 This step is only applicable for Umbraco projects started on version 11 or below.
```

Comment:
the steps below are only for projects below v11 as this hint states. Perhaps in this case it might be better to change the hint style="info" to hint style="warning" to make this more obvious?

Reply:
Missed that! I was just sparked by this forum post

https://our.umbraco.com/forum/using-umbraco-and-getting-started/113733-the-delivery-api-is-not-enabled-no-indexing-will-performed-for-the-delivery-api-content-index

so was thinking ahh, where have they seen the advice to update startup.cs... so I was looking specifically for the sample on the V13 version of the documentation, rather than reading it linearly... so I missed the 'hint'

I think we used to struggle with the balance of this with V6/v7/v8 docs - you want 'only' the information for the version that the person has selected in the docs, but sometimes there is very little change between the articles and also what about people who aren't using the latest version...

... with developer and documentation - often people are quickly searching/skimming a page for an answer, their time is precious, it's different to sitting down to read - the code sample is 'the gold' it's where attention is drawn to, the developer brain understands the code better than the description (well not all devs, we all consume and learn differently)

Anyway, I think that is the trap the person in the forum fell into and I followed...

But what do we do for 'V12 and below users' who are accidentally visiting the V13 page documentation? when they haven't adopted V13 yet? and for how long do we support that journey as 'the default' given V13 is the LTS.

I think you could put the additional legacy information inside an accordion? now that we can do that in the docs??

so you'd have

### Register the Content Delivery API Dependencies

For V13, and above the Content Delivery API Dependencies are automatically registered via .AddDeliveryApi() in the solution's project.cs File

For V12 and Below [v]

(this is the bit inside the accordion and describes the steps for registering dependencies in startup.cs)

This way nobody is accidentally drawn to the code sample, but you are still supporting the swathes of people who haven't adopted V13 yet - also when you come to edit this page  for V17 LTS, you can think ok, we can remove this accordion now, and it's much easier to see what needs to be removed.

Reply:
I think that is a really good idea! So below this text "Umbraco projects started on version 11 or below also need to opt-in through code by [registering the delivery API dependencies](https://github.com/umbraco-cms/reference/content-delivery-api#register-the-content-delivery-api-dependencies). Below you will find a description of how to do this." add accordion with the steps under "Register the Content Delivery API dependencies for v12 or below". 👍 

Would you be up for making this change or would you prefer me to do it asap? 😊 

![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/bb5ed8a6-e87b-4590-913f-c12c101b29bc)

Reply:
HI Alina - I don't know how to do the magic accordion markup!!!

feel free to tweak and I can observe, or I may get chance over the weekend to have a look.

regards

Marc

Reply:
Hey Marc, you can find an example here: https://github.com/umbraco/UmbracoDocs/blob/main/14/umbraco-cms/tutorials/creating-a-property-editor/adding-configuration-to-a-property-editor.md?plain=1. Where it is written “see the entire file”  

Shortly it needs the details and summary tags as per the gitbook docs (gitbook being the platform we use for our docs) https://docs.gitbook.com/content-editor/blocks/expandable 

You can give it a try if you have some time and then I can review and help along the way 😊